indico/modules/events/abstracts/controllers/display.py

```diff
@@ -44,3 +49,26 @@ def _process(self):
         pdf = AbstractToPDF(self.abstract)
         file_name = secure_filename('abstract-{}.pdf'.format(self.abstract.friendly_id), 'abstract.pdf')
         return send_file(file_name, pdf.generate(), 'application/pdf')
+
+
+class RHDisplayProtectionBase(RHConferenceBaseDisplay):
```

Comment:
Having the page accessible even without the menu item enabled doesn't disclose anything - it only shows data *you* entered. I don't think we need this check here. We have it for the other listings because those show names of lots of people.